### PR TITLE
Updates to /talks

### PR DESCRIPTION
**Pull request by jasminwong**
This update makes changes to the talk schedule on /talks by making it a more explicit timeline and listing page with links to the speaker bios. Also updated is the homepage, where there is now a button under "Conference" that links to /talks.

<details>
<summary>Screenshots</summary>

talks

homepage-new-btn

</details>

### CODE REVIEW

File: data/talks.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  - timeslot: "09:00–09:30"
+    title: "Intro"
+    abstract: ""
+    slug: "intro"
+    speaker:
+      name: ""
+      avatar: ""
+  - timeslot: "09:30–10:00"
+    title: "Understanding 'Hello World'"
+    abstract: "'Hello, World!' is the first step of many a ruby journey. How much can we learn by going deep on everything it can teach us? Join me on a fun dive down through the stack as we go deep on everything from Strings, IO, Unix, the Ruby VM and x86 Assembler letting the simplest of programs be our guide."
```
Comment (mermop) [inline data/talks.yml]: Rather than duplicating these talks, titles & abstracts - how about we somehow cross reference them? So:

```
#talks.yml (maybe this should even be schedule.yml)
schedule_items:
  - timeslot: "08:30-09:00"
    title: "Registration"
    description: "Pick up your nametag and mingle at Te Papa."
    slug: "rego"
  - timeslot: "09:00-09:30"
    title: "Intro"
    description: "Introduction to the conference."
    slug: "intro"
  - timeslot: "09:30–10:00"
    talk: "eoin"
    slug: "eoin"
```

Then in the view (although this much Ruby should probably go somewhere else...)

```
schedule_items.each do |schedule_item|
  .c-talk id=schedule_item.slug
    .c-talk__timeslot.col-1-4 = schedule_item.timeslot
    .c-talk__detail.col-3-4
      - if timeslot.talk
        - talk = data.speakers.speaker_list.find { |speaker| speaker.slug == timeslot.talk }
          h2.c-talk__title = talk.title
          p.c-talk__abstract= talk.abstract
          .c-talk__speaker
             img.c-talk__speaker-avatar src=("/images/speaker-avatars/#{talk.speaker.avatar}") alt=(talk.speaker.name)
          h3.c-talk__speaker-details 
            a href="/speakers/##{talk.slug}"
              = talk.speaker.name 
      - else
        h2.c-talk__title = schedule_item.title
        h2.c-talk__abstract = schedule_item.description
```

Then you've got the speaker data from the speaker data file and we don't have to double up on anything. It would break if someone put something that wasn't referenced in a `talk` but we could address that with some more `if` statements/I'm not too concerned

File: source/partials/_head_big.html.slim [hunk 2]
```diff
@@ -5,8 +5,7 @@
         img.hero__logo alt="Kiwi Ruby - New Zealand Ruby Conference 2017" src="./images/kiwiruby.svg"
       .hero__slogan
         span.hero__hidden Kiwi Ruby
-        br
         p.hero__date
           ' November 2-3 2017
-        p.hero__date
+          br
```
Comment (mermop) [inline source/partials/_head_big.html.slim:10]: Why this change? Seems unrelated

**Review by mermop — CHANGES_REQUESTED** [review]
Looks good - in terms of the structure I think there's too much doubling up between `speakers.yml` and `talks.yml` - have made one reasonably big suggestion but I think it shouldn't be too hard to implement

File: source/partials/_head_big.html.slim (hunk 2 above)
Comment (jasminwong) [reply source/partials/_head_big.html.slim:10]: @mermop It is unrelated, yes! While I was there I thought to squish the hero date and location together under the logo on desktop, the line spacing between them seemed too high.

File: data/talks.yml (hunk 1 above)
Comment (jasminwong) [reply data/talks.yml]: @mermop Thanks for the suggestion. I've changed around the yaml structures, and with Tim's help have refactored the view to pull data from `speakers` and `schedule`.  It's ready to go now once you've had a look. Thanks for your patience!

**Comment by mermop** [comment]
The changes look good - I wonder about putting a link in to the after party after #35

**Review by mermop — APPROVED** [review]
